apache/nuttx
# board/stm32f401rc-rs485: Add support to Mass Storage

## Summary
Add support to USB Mass Storage

## Impact
Users will be able to use USB Mass Storage

## Testing
- Configure using "usbmsc" configuration
- Build and flash
- Connect to NSH and run msconn
- A USB Mass Storage should be enumerated on your machine.
